views/conditions/type-2-diabetes/finding-help-and-support.html

```diff
@@ -0,0 +1,153 @@
+{% extends "_layouts/_layout-default.html" %}
+
+{% set pagetitle = "Finding help and support - Type 2 diabetes - - NHS.UK Alpha" %}
+
+{% block localheader %}
+  <div id="local-header">
+
+    <div class="header-local-container">
+        <h1>Type 2 diabetes</h1>
+          
+          <nav class="condition-navigation">
+          <ol class="nav-list">
+              <li class="nav-item">
+                  <a href="check-if-you-have-it">Check if you have it</a>
+              </li>
+              <li class="nav-item">
+                  <a href="getting-diagnosed">Getting diagnosed</a>
+              </li>
+              <li class="nav-item">
+                  <a href="understanding-medication">Understanding medication</a>
+              </li>
+              <li class="nav-item">
+                  <a href="going-for-regular-check-ups">Going for regular check ups</a>
+              </li>
+              <li class="nav-item">
+                 <a href="health-problems">Health problems</a>
+              </li>
+              <li class="nav-item">
+                 <a href="food-and-keeping-active">Food and keeping active</a>
+              </li>
+              <li class="nav-item">
+                  Finding help and support
+              </li>
+          </ol>
+
+    </div>
+
+
+  </div>
+{% endblock %}
+
+{% block content %}
+
+<div class="content">
+
+    <div class="grid-row">
+
+      <div class="grid-2-3">
+
+          <h2 class="heading-xlarge" id="finding-help-and-support-for-type-2-diabetes">Finding help and support for type 2 diabetes</h2>
```

Comment:
It was a gross copy and paste from the original files. There may have been logic there that I didnt want to arbitrarily over-write.. ;-)